packages/SketchMorph2-Core.package/M2Backend.class/instance/drawCircle.st

```diff
@@ -5,4 +5,5 @@ drawCircle
 		isErasing: false;
 		mode: #pen;
 		currentTool: #drawCircle;
+		currentCursor: (Form dotOfSize: self brushSize color: self penColor);
```

Comment:
Maybe we should add a cursor for that like we did for `drawEraser` to make the code more consistent.